addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -238,7 +238,7 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             if ($(ev.currentTarget).data('showResults')) {
                 document.location = `/survey/results/${encodeURIComponent(self.surveyId)}`;
             } else {
-                window.history.back();
+                document.location = `/web#view_type=form&model=survey.survey&id=${encodeURIComponent(self.surveyId)}&action=survey.action_survey_form`;
```

Comment:
I don't think it's right... If we're in another tab (not talking about the tour), why not instead show 'Session ended. You can now close this tab"?